src/T2DMap.cpp

```diff
@@ -2075,17 +2077,24 @@ void T2DMap::paintEvent(QPaintEvent* e)
             // If one or more rooms are selected - make the text slightly orange.
             switch (selectionSize) {
             case 0:
-                infoText.append(
-                        tr("Room ID: %1 (Current) Position on Map: (%2,%3,%4)\n").arg(QString::number(roomID), QString::number(_prid->x), QString::number(_prid->y), QString::number(_prid->z)));
+                infoText.append(tr("Room ID: %1 Position on Map: (%2,%3,%4) {current Player location}\n",
```

Comment:
```suggestion
                infoText.append(tr("Room ID: %1 Position on Map: (%2,%3,%4, current player location)\n",
```

Reply:
I don't think anyone uses {} in UIs in England, though?

Reply:
I wanted to use something that didn't look like the coordinates - to keep them as *visually distinct* elements. Also we (I) uses `{`...`}` extensively in the Lua error messages when they explain a default or some extra detail inside the explanation in `(`...`)` of what type a supplied argument should have been and what it was...

Reply:
Removed the second set of brackets/braces and just separated the text from the previous part with ` - `.